unit_test/test_spdm_requester/challenge.c

```diff
@@ -3889,6 +3974,83 @@ void libspdm_test_requester_challenge_case28(void **state)
     free(data);
 }
 
+/**
+ * Test 27: The key usage bit mask is not set, the SlotID fields in CHALLENGE and CHALLENGE_AUTH shall not specify this certificate slot
+ * Expected Behavior: get a LIBSPDM_STATUS_INVALID_MSG_FIELD return code
+ **/
+void libspdm_test_requester_challenge_case27(void **state)
```

Comment:
`libspdm_test_requester_challenge_case27` should be `libspdm_test_requester_challenge_case29`?